prow/cmd/deck/static/prism.js

```diff
@@ -0,0 +1,12 @@
+/* PrismJS 1.15.0
+https://prismjs.com/download.html#themes=prism-coy&languages=bash+git+yaml+vim&plugins=line-numbers+toolbar+highlight-keywords+command-line+copy-to-clipboard */
```

Comment:
License? looks like this is MIT which requires including the license https://github.com/PrismJS/prism/blob/master/LICENSE